shared/actions/config/index.js

```diff
@@ -120,15 +120,16 @@ export function bootstrap (): AsyncAction {
   return (dispatch, getState) => {
     if (!bootstrapSetup) {
       bootstrapSetup = true
-      console.log('Registered bootstrap')
+      console.log('[bootstrap] registered bootstrap')
       engine().listenOnConnect('bootstrap', () => {
-        console.log('Bootstrapping')
+        console.log('[bootstrap] bootstrapping on connect')
         dispatch(bootstrap())
       })
     } else if (getState().dev.reloading) {
       // Let's still register the listeners
       dispatch(_registerListeners())
     } else {
+      console.log('[bootstrap] performing bootstrap...')
```

Comment:
I wanted this to differentiate whether we hit the `bootstrapSetup` branch vs. actually bootstrappging. Do you think it is superfluous?